core/nuxeo-operation.js

```diff
@@ -75,6 +75,16 @@ import './nuxeo-page-provider.js';
           notify: true,
         },
 
+        /** Indicates if the request behind this operation cannot be cancelable.
+         * By default a request is cancelable, it means if on the same `nuxeo-operation`
+         * we perform two sequential requests: Request A & Request B. The first one will be
+         * aborted and we keep the last one (in our case request B). This is done to avoid
+         * an obsolete responses.
+         * */
+        notCancelable: {
```

Comment:
Despite not being perfect, for readability purposes I would suggest to call the property `uncancelable`

Reply:
Despite that 👍